src/Sarif.FunctionalTests/SarifConverterTestData/FxCop/FxCopExampleReport2.xml.sarif

```diff
@@ -17,20 +17,60 @@
           }
         ]
       },
+      "logicalLocations": {
+        "SecurityCryptographyRuleTests": [
+          {
+            "name": "SecurityCryptographyRuleTests",
+            "kind": "namespace"
+          }
+        ],
+        "securitycryptographyruletests.dll": [
```

Comment:
is this a module or an assembly? multiple modules can be linked into a single binary